samples/wifi/radio_test/src/nrf_wifi_radio_test_shell.c

```diff
@@ -832,10 +868,12 @@ static int nrf_wifi_radio_test_set_tx_power(const struct shell *shell,
 
 	val = strtoul(argv[1], &ptr, 10);
 
-	if (val > 24) {
+	if (((val > MAX_TX_POWER_FOR_RADIO_TEST) &&
+		(val < MAX_TX_POWER_FOR_SYSTEM_TEST)) ||
+		(val > MAX_TX_POWER_FOR_SYSTEM_TEST)) {
```

Comment:
Alignment seems to be off. Also this could be changed to:
`if (((val > MAX_TX_POWER_FOR_RADIO_TEST) &&
      (val != MAX_TX_POWER_FOR_SYSTEM_TEST))) {`